spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/listener/QueueAttributesResolver.java

```diff
@@ -91,9 +95,18 @@ private CompletableFuture<String> resolveQueueUrl() {
 	}
 
 	private CompletableFuture<String> doResolveQueueUrl() {
-		return CompletableFutures
-			.exceptionallyCompose(this.sqsAsyncClient.getQueueUrl(req -> req.queueName(this.queueName))
-				.thenApply(GetQueueUrlResponse::queueUrl), this::handleException);
+		Arn arn = convertToQueueArn(this.queueName);
```

Comment:
Minor nit: why pass `this.queueName` as an argument? We could simply use it in the method right?

Reply:
Definitely, right! I have changed that as well.